Library/Formula/mpv.rb

```diff
@@ -0,0 +1,72 @@
+class Mpv < Formula
+  desc "Video player based on MPlayer/mplayer2"
+  homepage "https://mpv.io"
+  url "https://github.com/mpv-player/mpv/archive/v0.13.0.tar.gz"
+  sha256 "1372704fd8f5701ef7d60f347fd15fe536e23ef148279bf4a4415e72896912db"
+  head "https://github.com/mpv-player/mpv.git"
+
+  option "with-libmpv",          "Build shared libray."
```

Comment:
`with-shared` would be a more consistent name. Also `library` rather than `libray`.